### PR TITLE
Update rails admin config to include week of response.

### PR DESCRIPTION
**Pull request by eschlange**
* Week of response relative to assignment creation date added to admin view.
* Added "Start Date" as a new User Date field.

[#103206552]

### CODE REVIEW

File: config/initializers/rails_admin.rb [hunk 1]
```diff
@@ -464,3 +486,21 @@ def answer
 
 
 end
+
+DAYS_IN_WEEK = 7
+
+def week_difference(start_date, end_date)
```
Comment (ericcf) [inline config/initializers/rails_admin.rb]: Should these methods be moved to the `Response` class?

**Comment by ericcf** [comment]
* Is there any reason not to delete the commented lines in the `app/models/user.rb` file?
* Should there be a validation on `start_date`?

File: config/initializers/rails_admin.rb (hunk 1 above)
Comment (eschlange) [inline config/initializers/rails_admin.rb]: Yes! This makes perfect sense. Also, this will allow me to write specs for these utility methods.

**Comment by eschlange** [comment]
I'll remove the commented files from ```app/models/user.rb```. They are not being used and do not seem useful. 

With regards to the start_date validation: this field is not required, it is for admin reporting use only and will not be set for some of the older participants. Is there any validation that you recommend for this type of field?

**Comment by ericcf** [comment]
Not necessarily, but sometimes it helps to have validation in place as a sanity check. If this won't impact the collected data then I guess it's not big deal.

**Comment by eschlange** [comment]
@ericcf I moved the response specific logic into the model, added specs, and cleaned up the admin config. It seems much more clean.

**Comment by ericcf** [comment]
Looks better to me! :fountain:

**Pull request merged**